runtime/gc_vlhgc/VLHGCAccessBarrier.cpp

```diff
@@ -298,47 +338,84 @@ MM_VLHGCAccessBarrier::jniGetPrimitiveArrayCritical(J9VMThread* vmThread, jarray
 	return data;
 }
 
+void
+MM_VLHGCAccessBarrier::copyBackArrayCritical(J9VMThread *vmThread, GC_ArrayObjectModel *indexableObjectModel,
+					J9InternalVMFunctions *functions, void *elems,
+					J9IndexableObject **arrayObject, jint mode)
+{
+	if (JNI_ABORT != mode) {
+		I_32 sizeInElements = (I_32)indexableObjectModel->getSizeInElements(*arrayObject);
+		indexableObjectModel->memcpyToArray(*arrayObject, 0, sizeInElements, elems);
+	}
+
+	/**
+	 * Commit means copy the data but do not free the buffer.
+	 * All other modes free the buffer.
+	 */
+	if(JNI_COMMIT != mode) {
```

Comment:
nitpick:
```suggestion
	if (JNI_COMMIT != mode) {
```
If isn't a function all so there should be a space between it and the `(`